versions-common/src/main/java/org/codehaus/mojo/versions/api/DefaultVersionsHelper.java

```diff
@@ -184,7 +185,12 @@ public ArtifactVersions lookupArtifactVersions(
                                     mavenSession.getRepositorySession(),
                                     new VersionRangeRequest(
                                             toArtifact(artifact)
-                                                    .setVersion(versionRange != null ? versionRange.toString() : "(,)"),
+                                                    .setVersion(ofNullable(versionRange)
+                                                            .map(VersionRange::getRestrictions)
```

Comment:
Here we're using the restriction present in the Version range object **if it's present** instead of the concrete version.